test/test_attributes.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright (c) 2025 National Land Survey of Finland (Maanmittauslaitos)
+#
+#  This file is part of geogen-algorithms.
+#
+#  This source code is licensed under the MIT license found in the
+#  LICENSE file in the root directory of this source tree.
+
+import geopandas as gpd
+import pytest
+from shapely.geometry import Polygon
+
+from geogenalg import attributes
+
+
+@pytest.mark.parametrize(
+    ("source_gdf", "target_gdf", "expected_attribute_value"),
+    [
+        (
+            gpd.GeoDataFrame(
+                {"id": [1], "mtk_id": ["3491230"]},
+                geometry=[Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])],
+                crs="EPSG:3067",
+            ),
+            gpd.GeoDataFrame(
+                geometry=[Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])],
```

Comment:
Target dataframe should have own attributes here, for example `{"some_attribute": ["a value"]}`. Then in the actual test code you could assert that the `result` does not contain "some_attribute" (`assert "some_attribute" not in result)